### PR TITLE
v2.6.0 release

### PR DESCRIPTION
**Pull request by nshoes**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## [2.5.3] - 2024-10-09
```
Comment (jjcarstens) [inline CHANGELOG.md:3]:
```suggestion
## [2.5.3] - 2024-12-03
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
...
+
+* Updated
+  * Match device_api_version to mix release version (#252)
```
Comment (jjcarstens) [inline CHANGELOG.md:6]: This also includes all the extension work, right? It probably warrants more comments and maybe even a minor bump?

Comment (nshoes) [reply CHANGELOG.md]: That's what I get for putting this together and talking to Digit at the same time 😆 Fixed.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## [2.6.0] - 2024-12-03
+This update includes Extensions, a mechanism to handle extensions on a device for custom data and/or reporting outside the firmware update mechanism. These are
+isolated from the socket as to not get in the way of potential firmware updates and can be defined in external libs as well as this one by implementing the `NervesHubLink.Extensions` behavior.
```
Comment (jjcarstens) [inline CHANGELOG.md:5]:
```suggestion
## [2.6.0] - 2024-12-03

This update includes Extensions, a mechanism to handle extensions on a device
for custom data and/or reporting outside the firmware update mechanism. These
are isolated from the socket as to not get in the way of potential firmware
updates and can be defined in external libs as well as this one by implementing
the `NervesHubLink.Extensions` behavior. Extensions are controlled by NervesHub
and can be adjusted at the product or device level for granular control.
```
Adjusted spacing, kept to 80 character line limit, and added note about NervesHub controlling

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,15 @@
...
+
+* Added
+  * Add support for Extensions using the Socket (#228)
```
Comment (jjcarstens) [inline CHANGELOG.md:8]:
```suggestion
  * Add support for Extensions using the Socket (#228)
    * Initial extensions supported are `NervesHubLink.Extensions.Health` and
      `NervesHubLink.Extensions.Geo`
```

Should we add a note about initial extensions supported?

**Review by jjcarstens — APPROVED** [review]

**Pull request merged**